datalakehouse-api/lakehouse_query_api.py

```
@@ -0,0 +1,558 @@
+
+
+# FIX: async, thread-pool dispatch, safe temp-file handling
+
+    # Parse JSON eagerly in the async thread so we fail fast before touching Spark
+    try:
+        payload_data = json.loads(request.payload)
+    except json.JSONDecodeError as e:
+        raise HTTPException(
+            status_code=status.HTTP_400_BAD_REQUEST,
+            detail={"status": "error", "code": 400, "message": "Invalid JSON payload"}
+        )
```

Ruff (0.15.10)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)